spec/qualys/qid_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "spec_helper"
+
+describe Qualys::Qid do
+  pending 'create some unit tests for the Qualys::Qid wrapper class'
+end
```

Comment:
If there's no immediate plans for this file just remove it. No point in setting TODO's no body plans to do.